dashboard/test/ui/features/applab/data_blocks.feature

```diff
@@ -24,3 +24,21 @@ Feature: App Lab Data Blocks
     And I open the debug console
     Then I press "runButton"
     And I wait until element "#successLabel" is visible within element "#divApplab"
+
+  Scenario: Start over clears Data
+    Given I am on "http://studio.code.org/s/allthethings/stage/18/puzzle/23?noautoplay=true"
+    And I rotate to landscape
+    And I wait for the page to fully load
+    And element "#runButton" is visible
+    Then I press "runButton"
+    And I wait until element "#keyValueLabel" is visible within element "#divApplab"
+    And element "#keyValueLabel" eventually contains text "success"
+    Then I press "versions-header"
+    And I wait until element "button:contains(Start over):eq(0)" is visible
+    And I click selector ".btn-danger"
+    And I click selector "#confirm-button" to load a new page
+    And I wait for the page to fully load
+    Then I add code "getKeyValue('key', function(value) {if (value === undefined) {textLabel('keyValueLabel', 'success')}});" to ace editor
+    And I press "runButton"
+    And I wait until element "#keyValueLabel" is visible within element "#divApplab"
+    And element "#keyValueLabel" eventually contains text "success"
```

Comment:
nice work! looks like it was not as hard as I suggested to add code via UI test steps. :-) I am wondering though, won't a `#keyValueLabel` containing `success` already be in the DOM even without this line, since the start code will be run again? or does the "add code" line overwrite that code? Just looking for a sanity check that this test will actually fail if the key-value pairs are not reset.